cypress-tests/cypress/support/commands.js

```diff
@@ -2365,26 +2310,38 @@ Cypress.Commands.add(
   }
 );
 
-Cypress.Commands.add("stepUp", (stepUpBody, globalState, value) => {
-  const key = `step_up_enabled_${globalState.get("merchantId")}`;
-  stepUpBody.key = key;
-  stepUpBody.value = value;
+Cypress.Commands.add("updateConfig", (configType, configData, globalState, value) => {
+  let url;
+  let body;
+  
+  switch (configType) {
+    case 'autoRetry':
+      url = `${globalState.get("baseUrl")}/configs/autoRetry`;
```

Comment:
Can you also define `key` within the case statements?

For `stepUp`: `const key = `step_up_enabled_${merchantId}`;`
For `maxRetries`: `const key = `max_auto_retries_enabled_${merchantId}`;`
For `autoRetries`: `const key = `should_call_gsm_${merchantId}`;`

And yes, this key should be passed in to the `url`. Something like: `${baseUrl}/configs/${key};`

Example:

```js
Cypress.Commands.add("updateConfig", (configType, configData, globalState, value) => {
  const base_url = globalState.get("baseUrl");
  const merchant_id = globalState.get("merchantId");

  let body;  
  let key;
  let url;

    case 'autoRetry':
	  key = `should_call_gsm_${merchant_id}`;
      url = `${baseUrl}/configs/${key}`;
```

Reply:
Done